src/nodes/Image.tsx

```diff
@@ -89,6 +83,19 @@ const uploadPlugin = options =>
     },
   });
 
+const IMAGE_CLASSES = ["half-right", "half-left"];
```

Comment:
I think it's worth making this a little more friendly to future extension and encode with a format like `left-50`, opening up to `left-25`, `center-50` etc in the future

Reply:
Sure, would happy to amend - thanks for reviewing.

The one risk with numbers is that they imply you can use any number and it might work (which could be a feature of course by writing the style more directly in the output).  We can also of course continue with words `quarter-left`.

What's the protocol here ?  Do we wait for @tommoor to advise ?

Reply:
I agree with Nan that this is a good idea for future extensibility. If the class was exposed to the end user I might worry about implying other options but it's not something they'll ever see so I'd be happy to just rename the ones we have 👍

Reply:
Perfect, will rename.